### PR TITLE
Update Zipkin remoteEndpoint preferences

### PR DESCRIPTION
**Pull request by MrAlias**
Do not reference semantic conventions that have been removed.

- The `net.peer.ip` attribute was removed: https://github.com/open-telemetry/opentelemetry-specification/pull/2614
  - Replace this with `net.sock.peer.addr` and or `net.sock.peer.name`
- The `http.host` attribute was removed: https://github.com/open-telemetry/opentelemetry-specification/pull/2469

### CODE REVIEW

**Review by pellared — APPROVED** [review]
Can you also update

https://github.com/open-telemetry/opentelemetry-specification/blob/025f8fc1a12dda35e42a25c96a0ca4e2c52e34c8/specification/logs/data-model.md?plain=1#L850

and 

https://github.com/open-telemetry/opentelemetry-specification/blob/025f8fc1a12dda35e42a25c96a0ca4e2c52e34c8/specification/logs/data-model.md?plain=1#L850

for consistency?

**Comment by MrAlias** [comment]
> Can you also update
> 
> https://github.com/open-telemetry/opentelemetry-specification/blob/025f8fc1a12dda35e42a25c96a0ca4e2c52e34c8/specification/logs/data-model.md?plain=1#L850
> 
> and
> 
> https://github.com/open-telemetry/opentelemetry-specification/blob/025f8fc1a12dda35e42a25c96a0ca4e2c52e34c8/specification/logs/data-model.md?plain=1#L862
> 
> for consistency?

I would rather scope this PR to updating the Zipkin remote endpoint preferences. Maybe that is a good change for another PR?

**Comment by pellared** [comment]
> I would rather scope this PR to updating the Zipkin remote endpoint preferences.

I am fine with this.

> Maybe that is a good change for another PR?

Let me know if you want to do it or I should create it.

**Comment by MrAlias** [comment]
> Let me know if you want to do it or I should create it.

Yeah, please create one. Thanks!

File: specification/trace/sdk_exporters/zipkin.md [hunk 1]
```diff
@@ -95,10 +95,10 @@ always available. The following table lists the possible attributes for
 |---|---|---|
 |1|peer.service|[OpenTelemetry adopted attribute for remote service.](../semantic_conventions/span-general.md#general-remote-service-attributes)|
 |2|net.peer.name|[OpenTelemetry adopted attribute for remote hostname, or similar.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
-|3|net.peer.ip & net.peer.port|[OpenTelemetry adopted attribute for remote address of the peer.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
-|4|peer.hostname|Remote hostname defined in OpenTracing specification.|
-|5|peer.address|Remote address defined in OpenTracing specification.|
-|6|http.host|Commonly used HTTP host header attribute for Http Spans.|
+|3|net.sock.peer.name|[OpenTelemetry adopted attribute for remote socket hostname of the peer.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
+|4|net.sock.peer.addr & net.peer.port|[OpenTelemetry adopted attribute for remote socket address of the peer.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
```
Comment (pellared) [inline specification/trace/sdk_exporters/zipkin.md:99]: I am looking at this PR the second time and I have lost confidence in this line.
I think that if `net.peer.name` is will not defined then probably we won't have `net.peer.port` as well. Therefore use should probably use `net.sock.peer.addr` instead.

```suggestion
|4|net.sock.peer.addr & net.sock.peer.port|[OpenTelemetry adopted attribute for remote socket address of the peer.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
```

File: specification/trace/sdk_exporters/zipkin.md [hunk 2]
```diff
@@ -95,10 +95,10 @@ always available. The following table lists the possible attributes for
 |---|---|---|
 |1|peer.service|[OpenTelemetry adopted attribute for remote service.](../semantic_conventions/span-general.md#general-remote-service-attributes)|
 |2|net.peer.name|[OpenTelemetry adopted attribute for remote hostname, or similar.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
-|3|net.peer.ip & net.peer.port|[OpenTelemetry adopted attribute for remote address of the peer.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
-|4|peer.hostname|Remote hostname defined in OpenTracing specification.|
-|5|peer.address|Remote address defined in OpenTracing specification.|
-|6|http.host|Commonly used HTTP host header attribute for Http Spans.|
+|3|net.sock.peer.name|[OpenTelemetry adopted attribute for remote socket hostname of the peer.](../semantic_conventions/span-general.md#general-network-connection-attributes)|
```
Comment (pellared) [inline specification/trace/sdk_exporters/zipkin.md:98]: You have forgotten to update https://github.com/open-telemetry/opentelemetry-specification/blob/638572cf21a94572a27810a29e463e24496388f9/specification/trace/sdk_exporters/zipkin.md?plain=1#L106

**Review by pellared — CHANGES_REQUESTED** [review]

**Comment by pellared** [comment]
@lmolkova PTAL

**Review by lmolkova — APPROVED** [review]
Thanks!

**Review by reyang — APPROVED** [review]

**Comment by carlosalberto** [comment]
@pellared Is your changes request still valid?

**Review by pellared — APPROVED** [review]

**Comment by pellared** [comment]
> @pellared Is your changes request still valid?

Comments addressed. Approved 👍

**Review by Kielek — APPROVED** [review]

**Comment by carlosalberto** [comment]
cc @yurishkuro

**Comment by yurishkuro** [comment]
tbh, I don't understand how to read that table. It gives OTEL names and OTEL definitions - how does that help in translating to Zipkin format? I would've expected either the equivalent Zipkin tag or the dedicated field in Zipkin data model.

**Comment by MrAlias** [comment]
> tbh, I don't understand how to read that table. It gives OTEL names and OTEL definitions - how does that help in translating to Zipkin format? I would've expected either the equivalent Zipkin tag or the dedicated field in Zipkin data model.

The table is an ordered ranking. This PR does not introduce or advocate for the table, it only updates semantic conventions. The table currently contains semantic conventions that were removed. This removes the stale values and replaces them with the replacement semantic conventions. It is an editorial change.

cc @carlosalberto

**Review by carlosalberto — APPROVED** [review]

**Pull request merged**